src/registrar/assets/js/get-gov.js

```diff
@@ -2799,12 +2802,18 @@ document.addEventListener('DOMContentLoaded', function() {
     if (radio != null) requestingSuborganization = radio?.checked && radio.value === "True";
     requestingSuborganization ? showElement(suborgContainer) : hideElement(suborgContainer);
     requestingNewSuborganization.value = requestingSuborganization && select.value === "other" ? "True" : "False";
-    requestingNewSuborganization.value === "True" ? showElement(suborgDetailsContainer) : hideElement(suborgDetailsContainer);
+    if (requestingNewSuborganization.value === "True") {
+      selectParent.classList.add("padding-bottom-2");
+      showElement(suborgDetailsContainer);
+    }else {
+      selectParent.classList.remove("padding-bottom-2");
+      hideElement(suborgDetailsContainer);
+    }
   }
 
   // Add fake "other" option to sub_organization select
   if (select && !Array.from(select.options).some(option => option.value === "other")) {
-    select.add(new Option("Other (enter your organization manually)", "other"));
+    select.add(new Option("Other (enter your suborganization manually)", "other"));
```

Comment:
**comment** I really don't think we should add content. We already have it in html files and .py files. It's okay to leave this as is, since it got through on the previous ticket. In general I would make an effort to use variables, where possible, when content is existing outside of an html file.

Reply:
@abroddrick  updated this with variables! Let me know what you think